templates/post_buildspec.yml.tpl

```diff
@@ -71,4 +77,16 @@ phases:
                         echo "Not managed development infrastructure"
                         echo "Subscribing log group "${APP_NAME}-${ENV_NAME}" to "$DATADOG_LAMBDA_FUNCTION_ARN""
                 fi
+        fi
+      - |
+        PIPELINE_TYPE=$(consul kv get "terraform/${APP_NAME}/app-env.json" | jq -r ".${ENV_NAME}.pipeline_type"  2>/dev/null)
```

Comment:
I see that we're doing this json parse on the fly each time we need something from consul.
Why dont we parse the entire json once in the pre-build step, and have a dict accesible for all subsequesnt requirements ? This Question is also for @Eli-Meitner .